app/src/main/java/com/dimagi/biometric/ParamManager.java

```diff
@@ -0,0 +1,154 @@
+package com.dimagi.biometric;
+
+import ai.tech5.finger.utils.SegmentationMode;
+
+/**
+ * Data class used for getting and setting various input parameters. Validation is done on setter
+ * methods to ensure that the values stays within constraints.
+ */
+public class ParamManager {
```

Comment:
These are all the available input parameters that a user can (optionally) give in their callout. Defaults are given as constants for when the user does not provide these values. When they do however, I think it might be good to ensure that the user's input is validated so that we do not get unexpected results when passing these params to Tech5's capture UI.